src/core/tsi/transport_security.h

```diff
@@ -65,7 +68,10 @@ struct tsi_handshaker_vtable {
   tsi_result (*create_frame_protector)(tsi_handshaker* self,
                                        size_t* max_protected_frame_size,
                                        tsi_frame_protector** protector);
+  /* Must be implemented by all TSI impls. */
   void (*destroy)(tsi_handshaker* self);
+  /* Methods for supporting the new async API.
```

Comment:
nit. This method supports both sync and async implementations.

Reply:
Changed wording to "async-capable".